src/extension/background-script/connectors/commando.ts

```diff
@@ -222,6 +247,54 @@ export default class Commando implements Connector {
       });
   }
 
+  async getTransactions(): Promise<GetTransactionsResponse> {
+    const incomingInvoicesResponse = await this.getInvoices();
+    const outgoingInvoicesResponse = await this.getPayments();
+
+    const transactions: ConnectorTransaction[] = [
```

Comment:
we do this merging in multiple connectors, should it be extracted to a utility function? e.g. `mergeTransactions(invoices,payments)`

Reply:
resolved